src/router.tsx

```diff
@@ -115,13 +115,13 @@ export const applicationMapNode: ApplicationMapType = [
         element: <MetricsPage />,
         loginNeeded: 'node',
       },
-      {
-        name: 'Configuration',
-        path: 'configuration',
-        icon: <SettingsIcon />,
-        element: <SettingsPage />,
-        loginNeeded: 'node',
-      },
+      // {
+      //   name: 'Configuration',
+      //   path: 'configuration',
+      //   icon: <SettingsIcon />,
+      //   element: <SettingsPage />,
+      //   loginNeeded: 'node',
+      // },
```

Comment:
Why was the node settings/configuration commented out from the router?

Reply:
Because it doesn't work, we get 404 when trying to update the settings. Disabling it for now